modules.json

```diff
@@ -67,12 +67,12 @@
                     },
                     "methyldackel/extract": {
```

Comment:
I'm confused why the modules were changed/updated here, did they get updated since https://github.com/nf-core/methylseq/commit/4aa0e789cc6fd5c524fd356eb4d46d4555113607 ?